explainaboard/analysis/feature_funcs.py

```diff
@@ -111,3 +139,19 @@ def cap_feature(s):
         return "first_caps"
     else:
         return "not_first_caps"
+
+
+def relative_position(sys_info: SysOutputInfo, text: str, word: str):
+    tokens = unwrap(sys_info.source_tokenizer)(text).strs
+    if word not in tokens:
+        return 0
```

Comment:
Is there any meaning of this particular value?
Since 0 seems to be also representing the first position, I think returning `None` would be better.

Reply:
I agree that 0 is not good. `None` would be best, but it would break our downstream processing, so I set it to "-1" for now.